cmd/kube-apiserver/app/testing/server_test.go

```diff
@@ -384,3 +385,65 @@ func crdExistsInDiscovery(client apiextensionsclientset.Interface, crd *apiexten
 	}
 	return false, nil
 }
+
+// TestOpenAPIPresence validates that various paths exist in the
+// complete delegation chain of the server. If the chain is not wired
+// up correctly this test should fail; this is largely a smoke test to
+// ensure that the complete delegation chain is complete.
+func TestOpenAPIPresence(t *testing.T) {
```

Comment:
`TestOpenAPIDelegationChainPlumbing` maybe better? And please s/validates/smoke tests/